src/go/rpk/pkg/cli/cmd/container/templates/env_vars.go

```diff
@@ -0,0 +1,17 @@
+package templates
+
+var SourceableEnvVars = `
```

Comment:
Probably worth moving this into the `container` package and making the variables private